conda-recipe/recipe.yaml

```diff
@@ -1,33 +1,33 @@
 context: {}
 package:
-
   name: molskill
-  version: '1.0'
+  version: '1.1'
 
 source:
   path: ../
 
 build:
-  number: 2
-
+  number: 1
```

Comment:
Some build number changed from `2` to `1`? (I'm not sure what this is, just highlighting something that surprised me, if it looks good then ignore my comment)

Reply:
Yes, by default it should be `1` but during debugging this locally I had changed this to `2`. `boa` uses this to differentiate between builds of the same version, but it is not automatically increased.